velox/functions/prestosql/tests/MapEntriesTest.cpp

```diff
@@ -125,6 +125,18 @@ TEST_F(MapEntriesTest, constant) {
   test::assertEqualVectors(expected, result);
 }
 
+TEST_F(MapEntriesTest, unknownValue) {
+  auto mapVector = makeMapVector<UnknownValue, UnknownValue>({
+      {},
+  });
+  auto result = evaluate("map_entries(c0)", makeRowVector({mapVector}));
+  auto keyVector = makeNullableFlatVector<UnknownValue>({});
+  auto valueVector = makeNullableFlatVector<UnknownValue>({});
+  auto elementVector = makeRowVector({keyVector, valueVector});
+  auto arrayVector = makeSingleRowArrayVector(elementVector);
```

Comment:
Can you also add a test related to your query : `SELECT MAP_ENTRIES(MAP(x,y)) FROM (VALUES (NULL,ARRAY[3])) AS t(x,y);` and ensure that map_entires returns null in this case.

Reply:
Added a test case for this below